test/Test/Prim/Int.purs

```diff
@@ -3,17 +3,32 @@ module Test.Prim.Int where
 import Prelude
 
 import Effect (Effect)
+import Test.QuickCheck (class Arbitrary)
+import Test.QuickCheck.Gen (chooseInt)
 import Test.QuickCheck.Laws (checkLaws)
 import Test.QuickCheck.Laws.Data as Data
 import Type.Proxy (Proxy(..))
 
+-- | Necessary for the EuclideanRing test
+-- | so as to prevent integer overflow when multiplying large integer values
```

Comment:
What do you think of naming this `SmallInt` instead, usable to avoid overflows when testing large integer values? It doesn't seem specific to `EuclideanRing` other than that we're using it in a test for it.

Reply:
Now that #57 is in, this could be refactored to use the `gen` version and not even need this newtype.